website/publish/domain_name.rst

```diff
@@ -81,9 +81,11 @@ We can now apply the redirection from your domain name's manager account:
 How to enable SSL (HTTPS) for my Odoo instance
 ==============================================
 
-To enable SSL, please use a third-party CDN service provider 
-such as CloudFlare.com. 
+Until recently, Odoo users needed to use a third-party CDN service provider such as CloudFlare to enable SSL.
 
+It is not required anymore: Odoo generates the certificate for you automatically, using integration with `Let's Encrypt Certificate Authority and ACME protocol <https://letsencrypt.org/how-it-works/>`__ (a separate certificate is generated for each domain name specified).
+
```

Comment:
I would add a "practical" word: "In order to get this, simply add your domain name in your customer portal."